src/dipdup/datasources/const.py

```diff
@@ -0,0 +1,7 @@
+class DatasourceKind:
```

Comment:
`kind` is just a hint for Pydantic to resolve unions in config correctly. It shouldn't be used anywhere except a single `Literal['kind']` typehint in classes that inherit `DatasourceConfig`.